src/Custom/Embeddings/Embedding.cs

```diff
@@ -83,6 +83,16 @@ internal Embedding(int index, BinaryData embeddingProperty, InternalEmbeddingObj
         Vector = ConvertToVectorOfFloats(embeddingProperty);
     }
 
+    // CUSTOM: Entirely custom constructor used by the Model Factory.
```

Comment:
Required to set `Vector` directly. The other constructor parses it from a `BinaryData` parameter.